src/qml/PowerPage.qml

```diff
@@ -19,55 +19,71 @@
 
 import QtQuick 2.9
 import org.asteroid.controls 1.0
-import org.asteroid.settings 1.0
 import Nemo.DBus 2.0
 
 Item {
 
     ListModel {
         id: powerModel
         //% "Power Off"
-        ListElement { text: qsTrId("id-poweroff-page"); icon: "ios-power-outline"; command: "req_shutdown" }
+        ListElement { text: qsTrId("id-poweroff-page"); icon: "ios-power-outline" }
         //% "Reboot"
-        ListElement { text: qsTrId("id-reboot-page"); icon: "ios-sync"; command: "req_reboot" }
+        ListElement { text: qsTrId("id-reboot-page"); icon: "ios-sync" }
+        //% "Reboot to bootloader"
+        ListElement { text: qsTrId("id-reboot-bootloader-page"); icon: "ios-bootloader-outline" }
```

Comment:
This will be a very long translation string and might not fully bring the point across for users.
I think it would be most useful to just call it "Fastboot Mode" or even only "Bootloader". Since thats what the user wants to achive, enter the fastboot bootloader mode.
The whole settings page is in context of "leaving the OS" and the list items just need to be named like the destination after powering down.
The icon still indicates that a reboot will happen to enter fastboot mode.

Reply:
That's a good point.  I also just noticed that my existing string doesn't even fit on the screen.  I've changed it to "Bootloader"